deployment/sriov-network-operator-chart/values.yaml

```diff
@@ -102,5 +106,7 @@ images:
   sriovDevicePlugin: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin
   resourcesInjector: ghcr.io/k8snetworkplumbingwg/network-resources-injector
   webhook: ghcr.io/k8snetworkplumbingwg/sriov-network-operator-webhook
+  metricsExporter: ghcr.io/k8snetworkplumbingwg/sriov-network-metrics-exporter
```

Comment:
now that we have chart push on release (tag) please adjust hack/release/chart-update.sh

Reply:
done! thanks for pointing out